File: problem_001.py
#!/usr/bin/env python

print("Project Euler - Problem 1")
print("Find the sum of all the multiples of 3 or 5 below 1000.\n")

number = 1
sum = 0

while number < 1000:
    if number % 3 == 0:
        sum += number
    else:
        if number % 5 == 0:
            sum += number
    number += 1

print("Sum: "+str(sum))

File: problem_001.js
#!/usr/bin/node

console.log("Project Euler - Problem 1:");
console.log("Find the sum of all the multiples of 3 or 5 below 1000.\n");

var sum = 0;
for (var n=3; n<1000; n++)
    if (n % 3 == 0 || n % 5 == 0)
        sum += n;

console.log(sum);
